docs/project-structure/genezio-configuration-file.md

```diff
@@ -629,6 +629,95 @@ frontend:
+
+- path: `string` **Required**
+
+  The path to the Dockerfile. It is relative to the `genezio.yaml` file.
+
+- environment: `Object` **Optional**
+
+  The environment variables that will be set for the server inside the Docker container.
+
+  You can use expression to define the environment variables. Check the [Usage](#expressions) section for more information.
+
+- timeout: `number` **Optional**
+
+  Specifies how long a request should wait for a response, in seconds. Default 30. [Pro Subscription](https://app.genez.io/billing) is required to change this configuration.
+
+- storageSize: `number` **Optional**
+
+  If this parameter is set, the execution environment will mount a storage disk with the specified size. The storage is not persistent across requests. Pro Subscription is required to change this configuration.
+
+- instanceSize: `tiny` | `medium` | `large` **Optional**
+
+  Determines the amount of RAM allocated to the execution environment. Default `tiny`. [Pro Subscription](https://app.genez.io/billing) is required to change this configuration.
+
+  tiny = 256MB RAM, 1 VCPU
+
+  medium = 384MB RAM, 1 VCPU
+
+  large = 512MB RAM, 1 VCPU
+
+  For larger instance sizes, [contact us](mailto:contact@genez.io).
+
+- maxConcurrentRequestsPerInstance: `number` **Optional**
+
+  Specifies the number of concurrent requests that can be served simultaneously by an execution environment. Default 10. [Pro Subscription](https://app.genez.io/billing) is required to change this configuration.
+
+### Example of `container` deployment configuration
+
+```yaml
+name: genezio-build-machine-backend
+region: eu-central-1
+yamlVersion: 2
+container:
+  path: ./Dockerfile
+  environment:
+    MY_ENV_VAR: my-value
+```
+
+## nextjs | nuxt | nitro
+
+You can use the `nextjs`, `nuxt`, or `nitro` field to deploy a Next.js, Nuxt.js, or Nitro project.
+
+- path: `string` **Required**
+
+  The path to the Dockerfile. It is relative to the `genezio.yaml` file.
+
+- environment: `Object` **Optional**
+
+  The environment variables that will be set for the server inside the Docker container.
+
+  You can use expression to define the environment variables. Check the [Usage](#expressions) section for more information.
+
+- `packageManager`: `npm` | `pnpm` | `yarn` **Optional**
+
+  The package manager used to install the project's dependencies. The default value is `npm`.
+
+- `scripts`: `Object` **Optional**
+
+The scripts that run before the deployment of the project. You can provide a list of commands that will be executed sequentially.
+
+Variables can be used in the scripts. Check the [Usage](#variables) section for more information.
+
+- `deploy`: `string` | `string[]` **Optional**
+
+  A general purpose script that runs before the project is deployed. Used to define how to install dependencies.
+
+### Example of `nextjs` deployment configuration
+
+```yaml
+nextjs:
```

Comment:
`nextjs`?

Reply:
Yup - I noticed we don't have nextjs, nitro, nuxt stuff and I added them in this PR.